announcements/2021/week08.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This week in provider-independent announcements (February 26, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Prometheus Operator v0.7.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.7.0/) upgrades to upstream chart v13.10.0, which contains Prometheus v2.24.0. (:warning: This is a breaking change and requires Helm 3 to install.)
+
+[Fluent Logshipping v0.6.0](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.0/) drops fluentd to reduce memory usage as fluent-bit now supports AWS outputs. It adds toleration to run on any node with taints.
```

Comment:
@yasn77 @QuentinBisson Please check if this is OK!

Reply:
Looks good 👍🏽

Reply:
ty!